kythe/go/serving/xrefs/columnar/columnar_test.go

```diff
@@ -121,24 +122,125 @@ func TestEncodingRoundtrip(t *testing.T) {
 	}}
 
 	for _, test := range tests {
-		kv, err := EncodeDecorationsEntry(nil, test)
-		if err != nil {
-			t.Errorf("Error encoding %T: %v", test.Entry, err)
-			continue
-		}
-		var file spb.VName
-		key, err := keys.Parse(string(kv.Key), &file)
-		if err != nil {
-			t.Errorf("Error decoding file for %T: %v", test.Entry, err)
-			continue
-		}
-		found, err := DecodeDecorationsEntry(&file, string(key), kv.Value)
-		if err != nil {
-			t.Errorf("Error decoding %T: %v", test.Entry, err)
-		} else if diff := cmp.Diff(test, found, ignoreProtoXXXFields); diff != "" {
-			t.Errorf("%T roundtrip differences: (- expected; + found)\n%s", test.Entry, diff)
-		}
+		t.Run(fmt.Sprintf("%T", test.Entry), func(t *testing.T) {
```

Comment:
Is this so the test invocation keeps going after the first failure?

Reply:
That, but it's mostly to name each subtest.